src/components/Forms/TextField.js

```diff
@@ -252,7 +259,7 @@ class TextField extends React.Component {
             type={type}
             aria-required={required}
             aria-invalid={hasError}
-            aria-describedby={hasError ? `error_${inputId}` : null}
+            aria-describedby={[hasError ? `error_${inputId}` : null, hintText ? `hint_${inputId}` : null].join(' ').trim()}
```

Comment:
Been having trouble getting this sort of logic to either be cleaner or follow complexity rules. Open to suggestions on refactoring this bit

Reply:
Not that much better, but here's another approach:
```
aria-describedby={[hasError && `error_${inputId}`, hintText && `hint_${inputId}`].filter(Boolean).join(' ')
```